packages/core/admin/admin/src/pages/MarketplacePage/tests/plugins.test.js

```diff
@@ -238,15 +209,13 @@ describe('Marketplace page - plugins tab', () => {
 
   it('filters a category option', async () => {
     const filtersButton = screen.getByTestId('filters-button');
-
     await user.click(filtersButton);
 
     const categoriesButton = screen.getByTestId('Categories-button');
     await user.click(categoriesButton);
 
     const option = screen.getByRole('option', { name: `Custom fields (4)` });
     await user.click(option);
-
     await waitForReload();
 
     const optionTag = screen.getByRole('button', { name: 'Custom fields' });
```

Comment:
`const optionTag = screen.getByText('Custom fields');`

Reply:
Unfortunately that's not precise enough because of the made by strapi icons on the plugin cards